design history posts (#364)
DFE-Digital/teacher-cpd-design-history: app/posts/ecf-v2/arrivals-transferring-in-an-ect.md

---

> title: 'Arrivals: transferring in an ECT and collecting programme choices'
> description: "How we've designed to transfer in an ECT from another school and asking for the ECT's programme choices."

Reviewer: ```suggestion
title: 'Registering an ECT who has been registered before and collecting their programme choices'
description: "How we've designed to transfer in an ECT from another school and asking for the ECT's programme choices."
```

date: 2025-02-28
---

## Why we’re doing this
We've designed a user journey which allows for schools to register ECTs at their school that have previously been registered at another school.

> We've tried to keep the journey as similar as possible to registering a new ECT. This is to keep the journeys as streamlined as possible, and for this reason we'll only have one call to action for a school to register an ECT.

Reviewer: Is this true with some of the homepage designs?

Is it more that we wanted to not expect schools to have to understand if an ECT had been registered at a previous school? That it would be too confusing/lead to schools getting it wrong?

## What we tested
We created and tested a journey which collected information about the ECT such as:
* name
* TRN
* start date
* working pattern

We also designed and tested playing back some information we'd already captured from the previous school, such as the lead provider and delivery partner, and asking the new school if this information would remain the same going forward.

![A screenshot of the initial design we tested to ask users if the programme choices would remain the same for the transferred ECT.](/ecf-v2/arrivals-transferring-in-an-ect/img.png)

However, after designing and testing this iteration, data analysis of the existing service found that transferred ECTs tend to take on the lead provider and delivery partner of the new school in the vast majority of cases.

The analysis found that in transfers of ECTs between schools in 2023:

> * 97% of ECTs were registered with the same lead provider as their new schools programme choice

Reviewer: ```suggestion
* 97% of ECTs were registered with the same lead provider as their new school's programme choices
```

> * 96% of ECTs were registered with the same delivery partner as their new schools programme choice

Reviewer: ```suggestion
* 96% of ECTs were registered with the same delivery partner as their new school's programme choice
```

> We then iterated our design to default to offering to register the ECT with the lead provider and delivery partner for the new school, rather than asking if they wish to retain the previous choices. We chose to still highlight that the ECT had been registered before, as we know that schools are reassured by us highlighting that we're aware of that.

Reviewer: ```suggestion
We then iterated our design to default to offering to register the ECT with the lead provider and delivery partner for the new school, rather than asking if they wish to retain the previous choices. We chose to still highlight that the ECT had been registered before, as we know that schools are reassured by us highlighting that we're aware of that. It confirms that the ECT has undergone training and induction before and that they can pick up where they left off.
```
